contributors.md

```diff
@@ -127,8 +139,24 @@ I also hope that it can convince people that don't consider themselves capable o
 > More recently that project has become consolidated into LA-CoNGA physics (http://laconga.redclara.net/), an EU Erasmus+ funded project with a mission to create a Latin American and European Community for Advanced Physics.
 > In this project I’m helping to build a data science module that will be thought in an online master course.
 
+<!---D--->
+
+
+<!---E--->
+
+
+<!---F--->
+
+
+<!---G--->
+
+
+<!---H--->
+
+
 ### Heidi Seibold
 
+* Role: Core contributor
```

Comment:
Is `Core contributor` kind of a catch all role? Are people who have specific tasks _also_ core contributors? (Needs some brainstorming I think - which is the work that you're doing now!)

Reply:
Sustained engagement is "core membership" and that's also where substantial contributions happen. But yes, there will a chapter on roles.